model_specific/SM/weinberg_angle.cpp

```diff
@@ -103,20 +66,20 @@ Weinberg_angle::Weinberg_angle()
    , precision_goal(1.0e-8)
    , rho_hat(0.)
    , sin_theta(0.)
-   , mw_pole(0.)
-   , data()
-   , susy_contributions(true)
 {
 }
 
-void Weinberg_angle::enable_susy_contributions()
-{
-   susy_contributions = true;
-}
-
-void Weinberg_angle::disable_susy_contributions()
+Weinberg_angle::Weinberg_angle(
+   const standard_model::Standard_model* model_,
+   const Sm_parameters& sm_parameters_)
+   : number_of_iterations(20)
```

Comment:
I think it would now be better to initialize these parameters (`number_of_iterations`, ..., `sin_theta`) in the `.hpp` file in order to avoid a redundant initialization in both constructors.

Reply:
Done in 55cfdda36c6fbec9c39c35c8b77385d59485f84e.